s4e-mac.test.c

```diff
@@ -1,6 +1,6 @@
 // Basic test of assembly and layout of sample S4E MAC instruction extensions
 
-// RUN: %clang --target=riscv32 -march=rv32ixs4emac -c -o %t.o %s
+// RUN: clang --target=riscv32 -march=rv32ixs4emac -c -o %t.o %s
```

Comment:
"Permissible hack" :-) - separately looking for explanation/doco of what %clang maps to and so why/when/where it should be used